src/ChargeTrapping/ChargeTrapping.jl

```diff
@@ -121,3 +121,85 @@ end
 
 
 
+
+"""
+    struct ConstantLifetimeChargeTrappingModel{T <: SSDFloat} <: AbstractChargeTrappingModel{T}
```

Comment:
In the documentation, you call it `ConstantLifetimeTrappingModel` without `Charge`. Please use consistent naming.